### DISCUSSION TITLE
Inputs/targets with missing values when using a sparse dummy regressor

### DISCUSSION THREAD

**Discussion by floresfred** [ourownstory/neural_prophet#716] [Q&A - get help using NeuralProphet]
I am running NP with AR lags, a single future regressor, and in global mode (i.e., 'ID' column in dataframe). After training I run NeuralProphet.predict() and get the **_ValueError: Inputs/targets with missing values detected. Please either adjust imputation parameters, or set 'drop_missing' to True to drop those samples_**.

First, I do not understand why this error arises at all. I pre-processed my dataset to ensure there are no missing values within a continuous daily time series.

The error seems related to my single future regressor which is a dummy variable (e.g., 1 or 0). The value of 1 occurs infrequently; only 6 or so times in a year with daily frequency (6/365 = 1.6% of the time). Is this the reason for the error? Is zero considered a missing value? If so how do I incorporate this type of regressor it into my model? It is an extremely important variable for me that explains spikes in the time series.

**Comment by wjh000123** [comment]
I don't know the background of it either, but after some investigation, we can pass `drop_missing=True` args when init NeuralProphet.
Code sample:

```python
m = NeuralProphet(
    yearly_seasonality=3,
    weekly_seasonality=False,
    daily_seasonality=8,
    growth="off",
    learning_rate=0.1,
    drop_missing=True
)
```
